Userland/Applications/TodoList/TaskListModel.h

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2021, Arthur Brainville <ybalrid@ybalrid.info>
+ *
+ * SPDX-License-Identifier: BSD-2-Clause
+ */
+
+#pragma once
+
+#include <AK/Vector.h>
+#include <LibGUI/Model.h>
+#include <LibGUI/Variant.h>
+
+#include "Task.h"
+
+#include <cstdio>
+
+class TaskListModel final : public GUI::Model {
+    AK::Vector<Task> task_list;
+
+public:
+    static AK::NonnullRefPtr<TaskListModel> create() { return adopt_ref(*new TaskListModel); }
+
+    virtual ~TaskListModel() override { }
+
+    virtual void update() override { }
+    virtual int row_count(const GUI::ModelIndex& = GUI::ModelIndex()) const override { return task_list.size(); }
+    virtual int column_count(const GUI::ModelIndex& = GUI::ModelIndex()) const override { return 1; }
+    virtual GUI::Variant data(const GUI::ModelIndex& index, GUI::ModelRole role = GUI::ModelRole::Display) const
```

Comment:
```suggestion
    virtual int column_count(GUI::ModelIndex const& = GUI::ModelIndex()) const override { return 1; }
    virtual GUI::Variant data(GUI::ModelIndex const& index, GUI::ModelRole role = GUI::ModelRole::Display) const
```

Please use east const. This also applies to other `const` qualifiers in your code.